From 1fe1eee353599c4fac47132872dc91abd0bb69aa Mon Sep 17 00:00:00 2001
From: Ivan Murashov <ivan.murashov@lge.com>
Date: Wed, 14 Dec 2022 19:31:37 +0000
Subject: [PATCH] IWYU: Add <cstring> for memcmp usage in general_names.cc

Otherwise build fails when building with use_custom_libcxx=false.
The error example:
net/cert/pki/general_names.cc:40:7: error:
use of undeclared identifier 'memcmp'; did you mean 'wmemcmp'?

Bug: 957519
Change-Id: I4db3751ac802ffdc5393f59a858a60ae1b39eba1
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4106310
Commit-Queue: Ivan Murashov <ivan.murashov@lge.com>
Reviewed-by: Victor Vasiliev <vasilvv@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1083263}
---

diff --git a/net/cert/pki/general_names.cc b/net/cert/pki/general_names.cc
index d2bbd25..4489c3916 100644
--- a/net/cert/pki/general_names.cc
+++ b/net/cert/pki/general_names.cc
@@ -4,6 +4,8 @@
 
 #include "net/cert/pki/general_names.h"
 
+#include <cstring>
+
 #include "net/cert/pki/cert_error_params.h"
 #include "net/cert/pki/cert_errors.h"
 #include "net/cert/pki/string_util.h"
